nodejs/week1/meals/src/backend/routes/large-meals.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const app = express();
+const meals = require("../data/meals.json");
+
+//	Respond with the json for all the meals that can fit lots of people
+app.get("/large-meals", (req, res) => {
+  res.send(meals.filter((meal) => meal.maxNumberOfGuests >= 7));
```

Comment:
These meals also need to include reviews

Reply:
I have included the reviews as well now